src/frontends/pytorch/src/op/addcmul.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <climits>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_addcmul(NodeContext& context) {
+    const auto eltwise_mult = std::make_shared<opset8::Multiply>(context.get_input(1), context.get_input(2));
+    const auto const_value = context.get_input(3);
```

Comment:
Does it mean that even if `value` is not specified in pytorch model it is still exist in graph as 1?